src/main/webapp/js/util.js

```diff
@@ -70,6 +70,24 @@ function waitClicked(e) {
 	}
 }
 
+function toggleVisibilityBasedOnRescan() {
+    // Get the checked state of the "Rescan" checkbox
+    var isRescanChecked = document.getElementsByName('rescan')[0].checked;
+
+    // Get the parent elements of the checkboxes to be hidden or shown
+    var scaCheckboxContainer = document.getElementById('includeSCAGenerateIRX').parentElement.parentElement;
+    var personalScanContainer = document.getElementById('personalScanCheckbox').parentElement.parentElement;
+
+    // Toggle visibility based on the "Rescan" checkbox state
+    if (isRescanChecked) {
+        scaCheckboxContainer.style.display = 'none';
```

Comment:
When the Rescan checkbox is selected, the Include SCA and Personal Scan options are hidden from the UI. Should we disable them instead so the user can still see the options but not edit them? Or, given that the Rescan option is selected, should we reduce distractions by hiding unnecessary fields?

Reply:
I would be ok with either option. What do we do with other options that are not relevant?